Author: Eric Maeker <eric.maeker@gmail.com>
Description: Remove sourceforge logo (privacy-breach-logo)
 privacy-breach-logo reported by `lintian -i`
Last-Update: 2014-07-14
Forwarded: no

--- a/doc/html/index.html
+++ b/doc/html/index.html
@@ -42,7 +42,7 @@
 </div><!--header-->
 <div class="contents">
 <div class="textblock"> 
-<a href="http://sourceforge.net"><img src="http://sourceforge.net/sflogo.php?group_id=142688&amp;type=7" style="width:210; height:62; border:none; float:right;" alt="Powered by SourceForge.net" /></a>
+<!-- privacy-breach-logo -->
  <h1><a class="anchor" id="overview"></a>
 Overview</h1>
 <p>QuaZIP is a simple C++ wrapper over <a href="http://www.winimage.com/zLibDll/minizip.html">Gilles Vollant's ZIP/UNZIP package</a> that can be used to access ZIP archives. It uses <a href="http://qt.digia.com/">the Qt toolkit</a>.</p>
